akka-stream-testkit/src/test/scala/akka/stream/testkit/StreamTestKitSpec.scala

```diff
@@ -184,11 +184,12 @@ class StreamTestKitSpec extends AkkaSpec {
     "#expectNextChainingPF should fail after timeout when element delayed" in {
       intercept[AssertionError] {
         val timeout = 100.millis
-        val overTimeout = timeout + (10.millis)
+        val overTimeout = timeout + 50.millis
```

Comment:
I don't get it, the old one failed because 10ms is the granularity of tick or something like that?

Reply:
I assumed that 10ms was quite tight. If `expectNextChainingPF` doesn't start its timeout within 10ms of the stream starting then it'll get the element and then fail

Reply:
Ah, thx for clarifying